docs/reference/api.rst

```diff
@@ -1,6 +1,31 @@
 Inmanta API reference
 =====================
 
+This page describes parts of the compiler that provide a stable API that could be used from modules.
+
+.. warning::
+    Only those parts explicitly mentioned here are part of the internal API. They provide a stable
+    interface. Other parts of the containing modules provide no such guarantees.
+
```

Comment:
```suggestion
    Only those parts explicitly mentioned here are part of the stable API. Other parts
    of the containing modules provide no such guarantees.

```
The term "internal API" confused me. "internal API" and "stable API" are pointing to the same thing here, so let's choose one term and use it consistently.

Reply:
I opted to just remove the "internal" keyword. I think the second sentence is required for the third to make sense.